Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,81 @@
+/*Global Styles*/
+
+body{
+	background-color:#F6F6F6;
+	padding: 75;
+	border-bottom: 5px solid #000;
+	border-top: 5px solid #000;
+	font-family: 'Lora', serif;
```

Comment:
As for the font question, I would do what you did here and apply the `font-family` to the `body` with the font that is used for the body copy so it is inherited by all of the elements, and then change that on a per-case basis. See me next note about the headings font.